net/fail2ban/Makefile

```diff
@@ -0,0 +1,106 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fail2ban
+PKG_VERSION:=0.11.2
+PKG_RELEASE:=1
+PKGARCH:=ALL
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/fail2ban/fail2ban
+PKG_MIRROR_HASH:=1899888a178e2a7d8b7234afcd96387559a7781fb4aa621c9288258e74753f47
```

Comment:
Why you are not using regular releases provided by GitHub? You might want to check how it is done e.g. in [netdata Makefile](https://github.com/openwrt/packages/blob/master/admin/netdata/Makefile)